pkg/webservice/handlers_test.go

```diff
@@ -1453,6 +1467,23 @@ func TestSpecificUserAndGroupResourceUsage(t *testing.T) {
 	assert.NilError(t, err, "Get User Resource Usage Handler request failed")
 	resp = &MockResponseWriter{}
 	getUserResourceUsage(resp, req)
+	assertUserExists(t, resp,
+		&dao.UserResourceUsageDAOInfo{
+			UserName: "testuser",
```

Comment:
`groupsLimitsConfig` does not define user `testuser`. why it can get passed?

Reply:
It looks like this is intentional. manager.go#getUserTracker returns the requested user tracker and creates one if it does not exist.
https://github.com/apache/yunikorn-core/blob/3b5a65d085ff49319e4ce6705e91a416efea1d70/pkg/scheduler/ugm/manager.go#L556-L570

Reply:
Ok, thanks